model/workflow.go

```diff
@@ -58,7 +58,7 @@ type BaseWorkflow struct {
 	// Key Domain-specific workflow identifier
 	Key string `json:"key,omitempty" validate:"omitempty,min=1"`
 	// Workflow name
-	Name string `json:"name,omitempty"`
```

Comment:
Is this removal intentional as the name is not mandatory?

Reply:
it's a typo, i will fix it.

Reply:
This field is not required.